package.json

```diff
@@ -22,5 +22,8 @@
     "fast-glob": "^3.2.4",
     "live-server": "^1.2.1",
     "nodemon": "^2.0.4"
+  },
+  "dependencies": {
+    "jszip": "^3.6.0"
```

Comment:
Do we need this since jszip is imported from CDN?

Reply:
Sorry for the mistake, I forgot to remove that part after I directly imported it from CDN.

Should I revert the change on my fork and re-make a pull request?